### PR TITLE
Add second arg head

### PR DESCRIPTION
**Pull request by calancha**
## What does this PR change?

Pass the right type for the first argument
Add a 2nd optional arg with the target from where to obtain the GPG keys ($server, default).

## Links

Fixes https://github.com/uyuni-project/uyuni/pull/2986
### Ports
- Manager-4.0: https://github.com/SUSE/spacewalk/pull/13393
- Manager-4.1: https://github.com/SUSE/spacewalk/pull/13392

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

### CODE REVIEW

**Review by lkotek — APPROVED** [review]
Thanks for fix :-)

**Comment by srbarrios** [comment]
Thanks @calancha, it is better passing directly the node as we do in the other functions indeed 👍

File: testsuite/features/step_definitions/common_steps.rb [hunk 1]
```diff
@@ -667,7 +667,7 @@
   # Prepare bootstrap script for different types of clients
   client = client_type == 'traditional' ? '--traditional' : ''
   node = get_target(host)
-  gpg_keys = get_gpg_keys(node)
+  gpg_keys = get_gpg_keys(node, target)
```
Comment (srbarrios) [inline testsuite/features/step_definitions/common_steps.rb:670]: Naming it is just a bit confusing now, as to obtain the node we use the function get_target, and then as second parameter of the next function we use the variable target extracted from few lines above 🤔
I just hope it is clear enough the next time we revisit this code

Comment (calancha) [reply testsuite/features/step_definitions/common_steps.rb:670]: I agree; there is room for improvement in the naming; it is worth improving it.

**Pull request merged**